devices/EFM32HG309F64.yaml

```diff
@@ -239,48 +209,18 @@ _add:
             access: read-only
       HFRCOCAL1:
         description: HFRCO calibration register 1
-        addressOffset: 0x180
-        size: 0x20
+        addressOffset: 0x30
+        size: 8
         fields:
           BAND21:
             description: 21MHz tuning value for HFRCO
             bitOffset: 0
             bitWidth: 8
             access: read-only
-      MEMINFO:
```

Comment:
Just glancing through, Should meminfo have been deleted?

Reply:
Right, it's not showing on the ref page, 
![Screenshot from 2019-12-14 12-35-49](https://user-images.githubusercontent.com/177826/70844063-24bfb200-1e6e-11ea-8a7e-0d97b3f57506.png)

So if it's there, I'm not sure what the actual offset is :thinking: